src/main/java/org/minions/devfund/mauricioz/linkedList/LinkedListUtils.java

```diff
@@ -0,0 +1,178 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+
+/**
+ * SD2x Homework #1
+ * Implement the methods below according to the specification in the assignment description.
+ * Please be sure not to change the signature of any of the methods!
+ */
+public final  class LinkedListUtils {
+    /**
+     * constructor.
+     */
+    private LinkedListUtils() {
+        return;
+    }
+    /**
+     * This method will add an element on a sorted list.
+     * @param list a list.
+     * @param value a value.
+     */
+    public static void insertSorted(final LinkedList<Integer> list, int value) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "list" object should be an interface such as "List" rather than the implementation "LinkedList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)